port_definition.go

```diff
@@ -0,0 +1,23 @@
+/*
+
```

Comment:
I noticed that (most) other files open up the copyright preamble with a `Copyright <year> Rohith All rights reserved` line. My knowledge about software licensing is at a bare minimum, so I'm just guessing that this might be relevant or totally not.

Pinging @gambol99 since he's the one mentioned on the line in question. :-)